core/src/main/java/edu/stanford/futuredata/macrobase/pipeline/BasicBatchPipeline.java

```diff
@@ -76,6 +78,15 @@ public BatchSummarizer getSummarizer(String outlierColumnName) throws MacrobaseE
                 summarizer.setMinRiskRatio(minRiskRatio);
                 return summarizer;
             }
+            case "fpgrowth": {
```

Comment:
prefer enums here, same as with @fabuzaid21... do you think it's better to use strings?